cmd/prometheus/config.go

```diff
@@ -190,45 +187,6 @@ func init() {
 		"Local storage engine. Supported values are: 'persisted' (full local storage with on-disk persistence) and 'none' (no local storage).",
 	)
 
-	// Remote storage.
-	cfg.fs.StringVar(
```

Comment:
Ok, after being even more confused, I now found this point in the blog post:

"Exact flag meanings may change as necessary. However, changes will never cause the server to not start with previous flag configurations."

I wonder if it was really intended like that for cases like this, but now that we guaranteed it, so be it.

So I'll re-add these flags, but with a big warning in the flag description and logged on startup?